src/SSLeay-standalone.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="SSLeay-standalone" name="SSLeay License - standalone">
+      <crossRefs>
+         <crossRef>http://www.openssl.org/source/license.html</crossRef>
```

Comment:
Found a cross-reference for this as a standalone: https://www.tq-group.com/filedownloads/files/software-license-conditions/OriginalSSLeay/OriginalSSLeay.pdf